src/applications/vaos/appointment-list/components/AppointmentsPage/tests/AppointmentCard.unit.spec.js

```diff
@@ -561,41 +561,6 @@ describe('VAOS Component: AppointmentCard', () => {
       },
     );
 
-    expect(await wrapper.findAllByText(/Friday, July 19/i)).to.exist;
-    expect(await wrapper.findAllByText(/12:00 p.m/i)).to.exist;
-  });
-  it('should display dates correctly', async () => {
```

Comment:
This is an almost identical duplicate test. I'm removing the one with fewer assertions.

Reply:
I think you're right, I'll double check before I remove them.